setup.cfg

```diff
@@ -1,3 +1,43 @@
+[metadata]
+name = distro
+version = 1.5.0
```

Comment:
Thoughts on closing https://github.com/python-distro/distro/issues/265 by using `version = attr: distro.__version__`?

Reply:
I addressed the issue in PR #294.

We can't use `version = attr: distro.__version__` as that would break the Sphinx configuration which parses `setup.cfg` to grab the version. I've taken the alternative approach of querying the installed version from `distro.py`. This is safer as other tools can continue to rely on setup.cfg as a source of truth.

Reply:
@sethmlarson I've applied this suggestion in the latest revision. 

@hartwork This also fixes your suggestion from earlier. The docs can now be built with Python 2 again.

More review feedback welcome.